gcp-deployment-manager/README.md

```diff
@@ -40,38 +33,121 @@ Replace `<keyname>` with something meaningful to identify the key, e.g. `docksal
 
-## Deploy the sandbox server
+### Local Setup
+
+You must have Docker installed. Local setup uses the image `google/cloud-sdk`. See the [image's Docker Hub page](https://hub.docker.com/r/google/cloud-sdk) for additional details on using this image.
+
+1. Run `make login`. This will ask you to copy/paste a URL into your browser and copy/paste a token. See [`gcloud auth login`](https://cloud.google.com/sdk/gcloud/reference/auth/login) for additional details.
+
+1. Run `make bash` for container CLI and run `gcloud config set project <project-id>`. Future `make`  commands will automatically run with the context of your project.
+
+### Deploying the sandbox
+
+After deployment you will have the following resources requisitioned (resources you pay for denoted by ($)):
+
+- VM instance (defaults to preemptible) ($)
+- Standard disk (150GB default) ($)
+- Static IP address ($)
+- Managed instance group
+- VM instance template
+
+Assuming you have already logged in and set the project as directed above, you simply need to run the following to setup your sandbox:
+
+    make create
+
+Then, in your GCP Console:
+
+1. navigate to Compute Engine > VM instances.
+1. Find the IP address assigned to your VM.
+1. Log into your VM via `build-agent@x.x.x.x`, where x.x.x.x is the IP identified in step 2.
+1. Run `fin system status`. If the Docksal system containers all appear as `Up` then everything is ready to go.
+1. If the Docksal system status is either not returned, or you get an error of some kind, you will want to: log out, wait a few minutes, then log back in. Run `fin system status` again and it should come back as up.
+
+**NOTE**: You will want to wait ~5 minutes after running `make create` to ensure the VM has had time to fully assemble itself. This is only true for first time setups.
+
+## Managing user access to the machine
+
+You can associate multiple SSH keys with the project to control user access, see [working with SSH keys](https://cloud.google.com/compute/docs/instances/adding-removing-ssh-keys) for details on how to add and remove SSH keys from the project.
+
+Project SSH keys, and their associated user, are automatically added to all VM instances in your project. If you add an SSH key to the project metadata with user id `user-a`, then after the machine has been provisioned, or restarted, that user will be able to login to the machine via `ssh user-a@x.x.x.x` where `x.x.x.x` is the sandbox VM's IP address.
+
+### Running commands as `build-agent`
+
+Sandboxes are stored in `/home/build-agent/builds`. Therefore, if you need to interact with a sandbox, you must be the user `build-agent`. All users with project-level SSH keys have passwordless sudo access. So, if you are logging into the machine as a user besides `build-agent`, to interact with the builds you must do the following:
+
+1. Run `ssh my-user-id@x.x.x.x`.
+1. Run `sudo -s` to assume root.
```

Comment:
This can be done as a one-liner: `sudo su - build-agent`

Reply:
Updated.